app/models/product.rb

```diff
@@ -1,3 +1,5 @@
 class Product < ApplicationRecord
   belongs_to :category
+  has_one_attached :image
+  scope :sort_by_name, ->{order name: :asc}
```

Comment:
```suggestion
  scope :sort_by_name, ->{order :name}
```
asc là default nên em lượt bỏ dc bé